colcon-build-and-test/action.yaml

```diff
@@ -49,7 +49,7 @@ runs:
 
     - name: Run rosdep install
       run: |
-        package_paths=$(colcon list -p --packages-above-and-dependencies ${{ inputs.target-packages }})
+        package_paths=$(colcon list -p --packages-above-and-dependencies ${{ inputs.target-packages }} --base-paths dependency_ws .)
```

Comment:
Thank you!

Could you fix other files as well?
https://github.com/autowarefoundation/autoware-github-actions/pull/68/files

Also, `dependency_ws` isn't created when `build_depends.repos` is empty, does this work correctly in that case?

Reply:
> Could you fix other files as well?

Sure!

> Also, dependency_ws isn't created when build_depends.repos is empty, does this work correctly in that case?

I confirmed the directory which doesn't exist does not cause any errors.
```
~/workspace/sandbox_ws/src/jetson_camera_trigger ros2
❯ ls
CMakeLists.txt  LICENSE    build_depends.repos  launch  package.xml  src
CPPLINT.cfg     README.md  include              log     setup.cfg

~/workspace/sandbox_ws/src/jetson_camera_trigger ros2
❯ colcon list -p --packages-above-and-dependencies jetson_camera_trigger --base-paths . dependency_ws
.
```